tests/trainers/tensorflow_examples.py

```diff
@@ -0,0 +1,53 @@
+"""Basic tests for the Python 3.6 and 3.9 base container images,based on tensorflow."""
```

Comment:
We also run these tests for Python 3.10 and in the future for other versions, I'd just omit the version here ;-)